integration_tests/tests/dbt_project.py

```diff
@@ -36,7 +44,22 @@ def get_dbt_runner(target: str) -> DbtRunner:
 
 class DbtProject:
     def __init__(self, target: str):
-        self.dbt_runner = get_dbt_runner(target)
+        self.dbt_project_copy_dir = Path(mkdtemp(prefix="integration_tests_project_"))
+        shutil.copytree(
+            DBT_PROJECT_PATH,
+            self.dbt_project_copy_dir,
+            dirs_exist_ok=True,
+            symlinks=True,
+        )
```

Comment:
Doing this to avoid some races that are happening in the CI
(I believe / suspect the random failures we're seeing are because multiple processes of dbt are running on the same folder and updating the manifest file - so now each test will have its own project folder)